### PR TITLE
adds applicantInfoConditional

### PR DESCRIPTION
**Pull request by lbai001**
This pr is for if the applicant is NOT veteran according to the issue description.
so far the titles of the fields are default titles without "Your", seems like that is still in discussion.

screencapture-localhost-3001-disability-benefits-apply-dependents-form-686c-dependents-applicant-information-part-2-2018-06-05-23_29_26

### CODE REVIEW

File: src/applications/disability-benefits/686/helpers.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const otherRelationshipDescription = (
 );
 
 export function isVeteran(item) {
```
Comment (jbalboni) [inline src/applications/disability-benefits/686/helpers.jsx]: Should we change the name of this function? It's now if you're not a veteran, not that you are a veteran.

File: src/applications/disability-benefits/686/config/form.js [hunk 2]
```diff
@@ -173,6 +168,23 @@ const formConfig = {
               }
             }
           }
+        },
+        applicantInformationConditional: {
```
Comment (jbalboni) [inline src/applications/disability-benefits/686/config/form.js]: We could call this `claimantInformation`, and have the path below be `claimant-information`.

Comment (lbai001) [reply src/applications/disability-benefits/686/config/form.js]: k

File: src/applications/disability-benefits/686/helpers.jsx (hunk 1 above)
Comment (lbai001) [reply src/applications/disability-benefits/686/helpers.jsx]: k

**Review by jbalboni — APPROVED** [review]

**Comment by emilywaggoner** [comment]
Thanks Annie! We will be using the "Your" pronouns here, so it looks like it will be in good shape as soon as those are in place. Thanks!

**Pull request merged**